How to use webpack

I cloned the repo. Ran npm i, tns install and then npm run start-android-bundle.
I get:

`ERROR in Error encountered resolving symbol values statically. Calling function 'ɵmakeDecorator', function calls are not supported. Consider replacing the function or lambda with a reference to an exported function, resolving symbol Injectable in /sample-Groceries/node_modules/nativescript-angular/node_modules/@angular/core/core.d.ts, resolving symbol ɵf in /sample-Groceries/node_modules/nativescript-angular/node_modules/@angular/core/core.d.ts, resolving symbol ɵf in /sample-Groceries/node_modules/nativescript-angular/node_modules/@angular/core/core.d.ts`

and

```
ERROR in ./main.aot.ts
Module not found: Error: Can't resolve './app.module.ngfactory' in '/home/lukas/WebstormProjects/sample-Groceries/app'
 @ ./main.aot.ts 3:0-60
```

along with a warning

```
WARNING in ../~/resolve-url-loader!../~/nativescript-css-loader!../~/nativescript-dev-webpack/platform-css-loader.js!./app.css
      resolve-url-loader cannot operate: CSS error
     /sample-Groceries/app/app.css:6:37: property missing ':'
      at error (/sample-Groceries/node_modules/css/lib/parse/index.js:62:15)
```
Can someone please hint what am I doing wrong.

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "@angular/platform-browser-dynamic": "~4.0.0",
     "@angular/router": "~4.0.0",
     "email-validator": "1.0.4",
-    "nativescript-angular": "1.5.0",
+    "nativescript-angular": "~1.5.1",
     "nativescript-iqkeyboardmanager": "1.0.1",
     "nativescript-social-share": "~1.3.2",
     "nativescript-unit-test-runner": "^0.3.3",

